test/prototype/test_low_bit_optim.py

```diff
@@ -42,7 +42,12 @@
     lpmm = None
 
 
-_DEVICES = ["cpu"] + (["cuda"] if torch.cuda.is_available() else [])
+if torch.cuda.is_available():
+    _DEVICES = ["cpu", "cuda"]
+elif torch.xpu.is_available():
+    _DEVICES = ["cpu", "xpu"]
+else:
+    _DEVICES = ["cpu"]
```

Comment:
(just a note) We don't have XPU CI testing, so this doesn't do anything for now. But still good for local testing.